Microsoft.DotNet.Interactive.Tests/PackageRestoreContextTests.cs

```diff
@@ -119,21 +143,29 @@ public async Task Can_get_path_to_nuget_packaged_assembly()
 
             var path = packageReference.AssemblyPaths.Single();
 
-            path.FullName
-                .ToLower()
-                .Should()
-                .EndWith("fluentassertions" + Path.DirectorySeparatorChar +
-                         "5.7.0" + Path.DirectorySeparatorChar +
-                         "lib" + Path.DirectorySeparatorChar +
-                         "netcoreapp2.0" + Path.DirectorySeparatorChar  +
-                         "fluentassertions.dll");
+            // path is a string similar to:
+            /// c:/users/someuser/.nuget/packages/fluentassertions/5.7.0/netcoreapp2.0/fluentassertions.dll
+            var name = path.Name;
+            var tfm = path.Directory.Name;
+            var reflib = path.Directory.Parent.Name;
+            var version = path.Directory.Parent.Parent.Name;
+            var packageName = path.Directory.Parent.Parent.Parent.Parent.Name;
+
+            name.ToLower().Should().Equals("fluentassertions.dll");
+            tfm.ToLower().Should().Equals("netcoreapp2.0");
+            reflib.ToLower().Should().Equals("lib");
+            version.ToLower().Should().Equals("5.7.0");
+            packageName.ToLower().Should().Equals("fluentassertions");
```

Comment:
This is a FluentAssertions gotcha. You're calling `Object.Equals` here, which won't cause the test to fail.

These should be, e.g., `packageName.ToLower().Should().Be("fluentassertions")`

Reply:
Sweet, thanks ...I will take care of that.